crates/bevy_ecs/src/hierarchy.rs

```diff
@@ -293,12 +293,49 @@ pub fn validate_parent_has_component<C: Component>(
     }
 }
 
+/// Returns a [`SpawnRelatedBundle`] that will insert the [`Children`] component, spawn a [`SpawnableList`] of entities with given bundles that
+/// relate to the [`Children`] entity via the [`ChildOf`] component, and reserve space in the [`Children`] for each spawned entity.
+///
+/// Any additional arguments will be interpreted as bundles to be spawned.
+///
+/// Also see [`related`](crate::related) for a version of this that works with any [`RelationshipTarget`] type.
+///
+/// ```
+/// # use bevy_ecs::hierarchy::Children;
+/// # use bevy_ecs::name::Name;
+/// # use bevy_ecs::world::World;
+/// # use bevy_ecs::children;
+/// # use bevy_ecs::spawn::{Spawn, SpawnRelated};
+/// let mut world = World::new();
+/// world.spawn((
+///     Name::new("Root"),
+///     children![
+///         Name::new("Child1"),
+///         (
+///             Name::new("Child2"),
+///             children![Name::new("Grandchild")]
+///         )
+///     ]
+/// ));
+/// ```
+///
+/// [`RelationshipTarget`]: crate::relationship::RelationshipTarget
+/// [`SpawnRelatedBundle`]: crate::spawn::SpawnRelatedBundle
+/// [`SpawnableList`]: crate::spawn::SpawnableList
+#[macro_export]
+macro_rules! children {
+    [$($child:expr),*$(,)?] => {
+       Children::spawn(($($crate::spawn::Spawn($child)),*))
```

Comment:
This still needs a `$crate` path:
```suggestion
       $crate::hierarchy::Children::spawn(($($crate::spawn::Spawn($child)),*))
```